README.md

```diff
@@ -554,6 +554,26 @@ await client.putFileContents("/my/file.txt", str);
 
 _`options` extends [method options](#method-options)._
 
+#### partialUpdateFileContents
+
+Update a remote file with a partial update. This method is useful for updating a file without having to download and re-upload the entire file.
+
+**_Note that this method is not standardised and may not be supported by all servers._** To use this feature, one of the following must be met:
+ * WebDav is served by Apache with the [mod_dav](https://httpd.apache.org/docs/2.4/mod/mod_dav.html) module
```

Comment:
Out of curiosity, where did you find evidence that Apache's mod_dav supports this feature ?

Reply:
Actually, from the sources of Apache:
https://github.com/apache/httpd/blob/c9028584a89a761e2804b596ce2c9f961f5b6fb8/modules/dav/main/mod_dav.c#L1094-L1108

And some discussions on other WebDAV related projects. (eg. https://github.com/miquels/webdav-handler-rs/blob/master/doc/Apache-PUT-with-Content-Range.md)